src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvoker.java

```diff
@@ -36,19 +35,27 @@
     private final CaseDataService caseDataService;
     private final CaseSanitiser caseSanitiser;
     private final SecurityValidationService securityValidationService;
+    private final AccessControlService accessControlService;
+    private final CaseAccessService caseAccessService;
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
we have some OM already configured that you could inject, have a look at: JacksonObjectMapperConfig